diff --git a/node_modules/@react-native-documents/picker/android/build.gradle b/node_modules/@react-native-documents/picker/android/build.gradle
index a41bc72..d7589d9 100644
--- a/node_modules/@react-native-documents/picker/android/build.gradle
+++ b/node_modules/@react-native-documents/picker/android/build.gradle
@@ -26,7 +26,7 @@ def isNewArchitectureEnabled() {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: "kotlin-android"
+// apply plugin: "kotlin-android"
 
 if (isNewArchitectureEnabled()) {
     apply plugin: "com.facebook.react"
@@ -34,7 +34,7 @@ if (isNewArchitectureEnabled()) {
 
 android {
     namespace "com.reactnativedocumentpicker"
-    compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
+    compileSdk getExtOrIntegerDefault('compileSdkVersion')
 
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
